spesifikasjoner/afpant/afpant-kjøperspantedokument/readme.md

```diff
@@ -1,21 +1,15 @@
-# Kjøpers pantedokument og følgebrev
- 
-* Overføring av 1 e-signert pantedokument (SDO) fra kjøpers bank til eiendomsmegler/oppgjørsforetak 
-* Overføring av følgebrev som PDF eller XML 
-* ACK/NACK-kvittering fra mottakersystem til avsendersystem med informasjon om forsendelsen kunne rutes korrekt
+# Løsning for elektroniske skjøtepakker
 
-For å kunne gjennomføre overføring av pantedokumentet er også bankens følgebrev inntatt i denne spesifikasjonen.
+Prosessen ved elektronisk tinglysing av skjøtepakker, innebærer at kjøpers bank oversender elektronisk signert pantedokument samt forutsetningsbrev til eiendomsmegler/oppgjørsforetak. Megler tinglyser deretter elektronisk skjøtepakke (e-signert skjøte og pantedokument) hos Kartverket.
```

Comment:
tanken er vel at kjøpers pantedokument/forutsetningsbrev alltid skal gå til meglerforetaket i Altinn (vi har forøvrig fortsatt på vår side oppgjørsfirma som mottar forsendelser). Uansett er det som regel oppgjørsforetaket som skal behandle/tinglyse kjøpers pantedokument sammen med skjøte (megler må sende videre til dem internt i systemet vårt f.eks) så jeg tenker det er helt greit at det står "eiendomsmegler/oppgjørsforetak".

Reply:
hm "Megler tinglyser deretter elektronisk skjøtepakke..."..når jeg tenker meg om så er det alltid oppgjørsforetaket som gjør dette hos oss hvertfall. Hvordan er det for andre leverandører? @tonjelund jeg lurer på om vi kanskje skulle vurdert å tatt en diskusjon rundt dette på et av de neste møtene våre :) For vår del hvertfall hadde det gitt mer mening at kjøpers pantedokument sendes til oppgjørsforetaket, men nå er vel tanken at alle dsve meldinger skal gå til meglerforetak (inkl kjøpers pantedokument). Kanskje det er det mest ryddige og jeg tror at det skapte problemer hos noen av leverandørene at oppgjørsforetak støttes som mottaker (har et vagt minne om noe her). Men vi burde kanskje diskutert på nytt litt ift oppgjørsforetak vs meglerforetak.

Reply:
Hm, ja vi tar med dette spørsmålet også i teknisk møte. Når det gjelder den korte beskrivelsen av tjenesten på github, så endrer jeg setn. foreløpig slik at det passer for begge scenarioene når både megler og oppgjør tinglyser.

"Prosessen ved elektronisk tinglysing av skjøtepakker, innebærer at kjøpers bank oversender elektronisk signert pantedokument samt forutsetningsbrev til eiendomsmegler/oppgjørsforetak, som deretter tinglyser elektronisk skjøtepakke (e-signert skjøte og pantedokument) hos Kartverket."